File: features/class_declaration_B/person_class.java
public class Person {

	public static void main(String[] args) {
		Person a = new Person("A");
		Person b = new Person("B");
	}
	
	private String name;
	
	public Person(String name) {
		this.name = name;
	}
	
    public String present() {
        return "Yes";
    }
	
	public String greet(String other) {  return other;  }
}

File: features/class_declaration_B/person_class.kt
class Person(private val name: String){
    fun present() : String = "Yes"
    fun greet(other: String) : String {
        return "Both $name and $other"
    }
}

fun main(args: Array<String>) {
    val A = Person("A")
    val B = Person("B")
}
